DependencyInjection/HttplugExtension.php

```diff
@@ -27,6 +27,13 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('services.xml');
         $loader->load('plugins.xml');
         $loader->load('discovery.xml');
+        if ($config['toolbar']['enabled'] && $container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
```

Comment:
It's always a good idea to inject logger. Fingers Crossed handler can
decide then when and what to log.

Tobias Nyholm <notifications@github.com> ezt írta (2016. január 8., péntek):

> In DependencyInjection/HttplugExtension.php
> <https://github.com/php-http/HttplugBundle/pull/19#discussion_r49206816>:
>
> > @@ -27,6 +27,13 @@ public function load(array $configs, ContainerBuilder $container)
> >          $loader->load('services.xml');
> >          $loader->load('plugins.xml');
> >          $loader->load('discovery.xml');
> > +        if ($config['toolbar']['enabled'] && $container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
>
> maybe look at what other bundles that plug into the toolbar do
>
> I've looked at a few. It is common that you inject a logger or a collector
> to a decorating class. That injection is done in a compiler pass. Some
> bundles (like GeocoderBundle
> <https://github.com/geocoder-php/BazingaGeocoderBundle> do always inject
> the logger, no matter debug or environment.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/php-http/HttplugBundle/pull/19/files#r49206816>.
>